tests/complete_run_v2_3_0.py

```diff
@@ -1,37 +1,41 @@
-import unittest
 import os
-from PIL import Image, ImageChops
-from system.test_diags import compare_images
-import sys
-sys.path.insert(0, '../examples')
-from tutorials.tutorial_v2_3_0_all_sets_E3SM_machines import run_lcrc
+import unittest
+
+from examples.tutorials.tutorial_v2_3_0_all_sets_E3SM_machines import run_lcrc
```

Comment:
`examples/tutorials` doesn't also need an `__init__.py` file? It's only for the top-level directories?

Reply:
Correct, you only need an `__init__.py` for the top-level directory to treat the entire directory as a module. 

`tutorials` will be a submodule of `examples` since it is a child directory, so you can do `import examples.tutorials`.